persons/search.html

```diff
@@ -1,81 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
-    <div class="main-content-block">
```

Comment:
We haven't worked on the Usaybia search form much if any, so if this form is already generic, no need to revert. But I couldn't tell if there is some Syriaca-specific stuff here?